### DISCUSSION TITLE
Moonray for 3ds Max?

### DISCUSSION THREAD

**Discussion by alexmurray2133** [dreamworksanimation/openmoonray#87] [Q&A] [answered]
Hello,

Is the USD implementation in Autodesk 3ds Max compatible or functional with Moonray when using the Hydra delegate or would we have to look at building a native 3ds max implementation?

I understand that Moonray must first be compiled on Windows prior to considering this a possibility.

Thank you!
Alex

**Comment by kjeffery** [comment, marked as answer]
You are correct in that MoonRay must first be ported to Windows. If 3ds Max has good Hydra support, it will work well with the caveat that MoonRay supports the UsdPreviewSurface material, but there will have to be further support to use the other MoonRay materials.
